src/io/IOManager.cpp

```diff
@@ -46,15 +46,12 @@ void IOManager::handleWorldState(proto::World & world) {
 }
 
 void IOManager::handleGeometry(proto::SSL_GeometryData & sslData) {
-  std::lock_guard<std::mutex> lock(geometryMutex);
-
-  // protobuf objects are not very long-lasting so convert it into an object which we can store way longer in field
-  FieldMessage msg = FieldMessage(sslData.field());
-  if (!SETTINGS.isLeft()) { msg.invert(); }
```

Comment:
I think this was the inversion message that didn't work? Is that the reason it is removed now? Did you test the functionality of inverting the field now?

Reply:
I tested the functionality of inverting the field and it does not have any meaning.